apache-jena/bin/tdb2.xloader

```diff
@@ -42,7 +42,7 @@ function testSortParallel() {
     set +e
     sort --parallel=3 < /dev/null &>/dev/null
     if [[ $? -ne 0 ]]; then
-        echo "No --parallel support in sort(1)" 2&>1
+        echo "No --parallel support in sort(1)" 2>&1
```

Comment:
@peutch Thanks for pointing this out.

It should be "write stdout message on stderr" -- `1>&2`

Done with #1788